source/localizable/routing/redirection.md

```diff
@@ -86,10 +100,10 @@ route's `beforeModel`, `model`, and `afterModel` hooks will fire again within
 the new, redirected transition. This is inefficient, since they just fired
 before the redirect.
 
-Instead, we can use the [`redirect()`][5] method, which will leave the original
+Instead, we can use the [`redirect()`][6] method, which will leave the original
 transition validated, and not cause the parent route's hooks to fire again:
 
-[5]: http://emberjs.com/api/classes/Ember.Route.html#method_redirect
+[6]: http://emberjs.com/api/classes/Ember.Route.html#method_redirect
```

Comment:
Inlined link identified by 6 instead. Now the numbering system is replaced with inline links, as @locks recommended. @toddjordan